embulk-input-jdbc/README.md

```diff
@@ -76,6 +77,38 @@ CREATE INDEX embulk_incremental_loading_index ON table (updated_at, id);
 
 Recommended usage is to leave `incremental_columns` unset and let this plugin automatically finds an auto-increment primary key. Currently, only strings and integers are supported as incremental_columns.
 
+### Use incremental loading with complex query
+
+**IMPORTANT**: This is an advanced feature and assume you have an enough knowledge about incremental loading using Embulk and this plugin
+
+Normally, you can't write your own query for incremental loading.
+`use_raw_query_with_incremental` option allow you to write complex query for incremental loading.
```

Comment:
The reason to use `use_raw_query_with_incremental` is writing optimized (faster) query, isn't it?
https://github.com/embulk/embulk-input-jdbc/issues/100#issuecomment-289352025

Reply:
Fixed at 9fe5f87b02e26b2f3b27b0cae7238b8d38e62c58 and added for embulk-input-(oracle|db2|sqlserver|postgresql|oracle|redshift) dc0f798c668eb44e2ced2075ddf5c882e3b48d3a